### DISCUSSION TITLE
Optionally don't include tasks in Daily Notes in "Today" column

### DISCUSSION THREAD

**Discussion by ItsFullOfCode** [roovo/obsidian-card-board#103] [Ideas]
- For tasks in Daily Notes: allow an option in Settings to **not** use the date of the Daily Note as the Due Date of the Task.
  - Use case: This feature would ally tasks in daily notes without an explicit due date to be shown in the "Undated" column. This would keep the "Today" column reserved for items who have an explicit due date for today or the past.

**Comment by roovo** [comment]
This is something I need to support for drag and drop functionality so I will look at this when I get on to that.  I'll create an issue (done it - #106) and put it on the dev board to do with some of the other drag and drop pre-features!

Thanks again :)
